website/docs/customize/policies/unique_password.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Password Uniqueness Policy
+sidebar_label: Password Uniqueness Policy
+support_level: authentik
+tags:
+    - policy
+    - password
+    - security
+    - enterprise
+authentik_version: "2025.4.0"
+authentik_enterprise: true
+---
+
+The Password Uniqueness Policy prevents users from reusing their previous passwords when setting a new password.
+
+## How it works
+
+This policy maintains a record of previously used passwords for each user. When a new password is created, it is compared against this historical log. If a match is found with any previous password, the policy is not met, and the user is required to choose a different password.
+
+The password history is maintained automatically when this policy is in use. Old password hashes are stored securely in authentik's database.
+
+:::info
+This policy takes effect after the first password change following policy activation. Before that first change, there's no password history data to compare against.
+:::
+
+## Configuration options
```

Comment:
@melizeche I *think we can actually remove this section, because both of those fields should be fully described down in your procedural steps, where they are creating the policy. If you want to leave this section, to highlight those two config settings, we can but let's pare it down to describe the fields but no go into great detail and duplicate what is in the steps. Thoughts?